client.py

```diff
@@ -73,6 +75,21 @@ def compare(row):
         return ret
     return sorted(state, key=compare, reverse=reverse)
 
+
+def create_bucket(config_startd_logging):
```

Comment:
I'm tempted to move this to another file.  This file is already long enough as it is.

Reply:
@dsschult Good point.  I'm moving this to client_util.py.